book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :title, :author
+  attr_accessor :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def persons
+    rentals.map(&:person)
+  end
```

Comment:
- [ ] Hey, kudos on incorporating the method for adding a book to the rentals array! 📚 However, the current setup misses establishing the crucial has-many relationship. 🤔 Take a cue from the example below for guidance on how to nail this aspect. 👇 Let's make sure our code forms those essential connections! 🚀💻

```suggestion
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
```